docs/details/data.dox

```diff
@@ -159,9 +159,31 @@ Requires that all dimensions except the join dimension must be the same for all
 
 \defgroup manip_func_tile tile
 
-\brief Tile the input array along specified dimensions
+\brief Repeat the contents of the input array along the specified dimensions
 
-Creates copys of the array a specified number of times within the output array
+Creates copies of the input array and places them side by side each other, such
+that the output array will have as many copies of the input array as the user
+specifies, along each dimension. In this way, the output array essentially
```

Comment:
I think you can shorten this `In this way, the output array essentially
becomes "tiled", wherein each copy of the input array is a "tile" (including the
input array itself).` further. 

Something like `Essentially, the output array will be a set of tiles where each tile is a copy of input array`.

Reply:
Cool that makes sense, I just mixed your statements with mine.